src/features/features.controller.ts

```diff
@@ -108,14 +190,39 @@ export class FeaturesController {
             );
             return updatedFeature;
         } else {
-            throw new HttpException("Forbidden", HttpStatus.FORBIDDEN);
+            throw new HttpException(
+                "user is unauthorized to perform this action",
+                HttpStatus.UNAUTHORIZED,
+            );
         }
     }
 
     @ApiOperation({
         summary:
             "Updates the order and category? of features by team members given a featureId (int), featureCategoryId (int), and order (int).",
     })
+    @ApiResponse({
```

Comment:
Small typo - can you remove the **_?_** in the summary for updateFeatureOrderAndCategory?